specificatie/openapi.yaml

```diff
@@ -648,6 +810,174 @@ components:
+        waarden:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Gebruiksdoel_enum'
+
+    AdresseerbaarObjectTypeZoekObject:
+      type: object
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:    
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/TypeAdresseerbaarObject_enum'
+    
+    PandenZoekObject:
+      allOf:
+        - $ref: '#/components/schemas/GeoJsonZoekOpdracht'
+        - type: object
+          description: Een object waarin de benodigde zoek criteria worden opgegeven om naar panden te zoeken.
+          properties:
+            status:
+              description: Een lijst met enumeratie waarden waar gezochte panden wel of niet aan moeten voldoen.
+              $ref: '#/components/schemas/PandStatusZoekObject'
+            geconstateerd:
+              description: Een indicatie of er naar geconstaeerde resources gezocht moet worden.
+              type: boolean
+            bouwjaar:
+              description: Een minimale en/of maximale waarde van het bouwjaar van paden waarnaar gezocht moet worden. 
+              $ref: '#/components/schemas/NumberOperator'
+
+    PandStatusZoekObject:
+      type: object
+      required:
+        - waarden
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/StatusPand_enum'
+
+    NumberOperator:
+      description: >-
+        "Een generieke aanduiding voor de min en max waarde van een attribuut van een object.
+        Als alleen een min waarde wordt opgegeven, dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan de min waarde.
+        Als alleen een max waarde wordt opgegeven, dan wordt er gezocht naar waarden die kleiner dan of gelijk zijn aan de max waarde.
+        Als zowel min en max waarde worden opgegeven waarbij min < max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die tussen min en max liggen.
+        Als zowel min en max waarde worden opgegeven waarbij min > max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die niet tussen min en max liggen." 
+      type: object
+      properties:
+        min:
+          type: number
+        max:
+          type: number
+
+    ArrayOperator:
+      type: string
+      description: >-
+        "Een lijst operatie, de volgende operaties worden ondersteund:
+        - and   : de gezochte resource property moet alle waarden uit de lijst bevatten
+        - or    : de gezochte resource property moet één of meer waarden uit de lijst bevatten
+        - not   : de gezochte resource property bevat geen van de waarde uit de lijst"
```

Comment:
Je kunt de volgende operaties gebruiken:
 - and   : de resource property moet alle waarden uit de lijst bevatten
 - or     :  de resource property moet één of meer waarden uit de lijst bevatten
 - not   :  de resource property mag geen van de waarden uit de lijst bevatten